chemie/events/urls.py

```diff
@@ -68,6 +68,17 @@
         "api/socialeventregistration/<int:pk>",
         views.SocialEventRegistrationDetail.as_view(),
     ),
+    path(
+        "påmeldingsgruppe/<int:pk>",
+        views.view_base_registration_group,
+        name="se_påmeldingsgruppe",
```

Comment:
Our standard is to have the `name="..."` in English. If you change this, be sure to change it if it is used in a template

Reply:
This is not uses anywhere else at the moment, so i simply changed the name to english.